source/projects/kleine-anfragen/de/index.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: project
-description: "Mit 'kleinen Anfragen' können Abgeordnete Informationen bei der Berliner Regierung anfragen. Wir haben diese Anfragen analysiert, um relevante Themen zu identifizieren und das darin verborgene Potenzial für Open Data zu ergründen."
+description: "Mit schriftlichen Anfragen können Abgeordnete Informationen bei der Berliner Regierung anfragen. Wir haben diese Anfragen analysiert, um relevante Themen zu identifizieren und das darin verborgene Potenzial für Open Data zu ergründen."
 lang: de
-title: "'Kleine Anfragen' analysiert"
```

Comment:
Just my 50 cent? Should be „Schriftliche Anfrage“ in DE “Schriftliche Anfrage” in EN

Reply:
Typographically correct … 🤓